api_client/python/timesketch_api_client/sigma.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Timesketch API client library."""
+from __future__ import unicode_literals
+
+import json
+import logging
+import pandas
+
+from . import error
+from . import index
+from . import resource
+
+logger = logging.getLogger('timesketch_api.sigma')
+
+
+class Sigma(resource.BaseResource):
+
+    def __init__(self, rule_uuid, api ):
+        """Initializes the Sigma object.
+
+        Args:
+            searchindex_id: Primary key ID of the searchindex.
+            searchindex_name: Name of the searchindex (optional).
+        """
+        self.rule_uuid = rule_uuid
+        self._resource_uri = 'sigma/{0:s}'.format(self.rule_uuid)
+        #self.sigma_data = self.lazyload_data()
+        super(Sigma, self).__init__(
```

Comment:
use an empty super call, as it should be in py3

```
super().__init__(api=api.....
```

Reply:
changed (but noticed we do other super calls in other methods, I just copied the code style